package.json

```diff
@@ -9,16 +9,28 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@code-hike/mdx": "^0.7.5-next.0",
+    "@mdx-js/loader": "^2.1.5",
+    "@mdx-js/react": "^2.1.5",
+    "@next/mdx": "^13.0.4",
+    "@types/mdx": "^2.0.3",
     "babel-plugin-styled-components": "^2.0.7",
     "bootstrap": "^5.2.2",
     "gray-matter": "^4.0.3",
+    "katex": "^0.16.3",
     "marked": "^4.1.1",
     "next": "13.0.0",
+    "next-mdx-remote": "^4.2.0",
     "react": "18.2.0",
     "react-bootstrap": "^2.5.0",
     "react-dom": "18.2.0",
     "react-multi-carousel": "^2.8.2",
     "react-responsive-carousel": "^3.2.23",
+    "rehype-highlight": "^6.0.0",
+    "rehype-katex": "^6.0.2",
+    "rehype-stringify": "^9.0.3",
+    "remark-frontmatter": "^4.0.1",
```

Comment:
Maybe you only need one: either this or `gray-matter`

Reply:
Actually, you're not using this package at all. Decide what you want to do :)

Reply:
I am using the first two of them, directly into the `serialize` method in `[slug].js`. 
With the others I was trying to make the markdown beautiful (changing fonts, code block's style and so on,)  but for now I have removed them because my trials didn't work.

Reply:
Yes, that's perfect. I was actually referring to just the last two :)

Reply:
Another thing I really want to do is to adopt the [`remark-gfm`](https://www.npmjs.com/package/remark-gfm#:~:text=4%20Versions-,remark%2Dgfm,-remark%20plugin%20to) package. 
I think it will be great to have the same `md` convention both on GitHub and into the webpage.

Reply:
I agree, but open a dedicated issue also for this one.

This PR is already doing something, so we can merge it on its own. Let's try to keep them as  atomic as possible: 1 PR -> 1 task :)